client/src/components/ApplicationDashBoard/ApplicationDashBoard.tsx

```diff
@@ -0,0 +1,84 @@
+import React, { useEffect, useState } from "react";
+import axios from "axios";
+import { useAppSelector } from "../../app/hooks";
+
+interface IStatusCount {
+  status: string;
+  count: number;
+}
+
+const ApplicationDashboard = (): JSX.Element => {
+  const [totalApplications, setTotalApplications] = useState(0);
+  const [applicationsByStatus, setApplicationsByStatus] = useState<
+    IStatusCount[]
+  >([]);
+  const [loading, setLoading] = useState(false);
+  const [error, setError] = useState<string | null>(null);
+  const user = useAppSelector((state) => state.user.userData);
```

Comment:
From this it looks like this feature is for a user to see their own stats, not anyone else's. Might be better to send request withCredentials and protect the route (get user from decoded cookie) rather than sending userId as a param

Reply:
100% Added a quick check to make sure the user info being requested matches the current user. This actually ties in to the ticket we discussed for EditProfilePage that fixes all the hangs. Code pushed.